static/app/views/performance/browser/webVitals/webVitalsDetailPanel.tsx

```diff
@@ -58,18 +61,17 @@ export function WebVitalsDetailPanel({
 }) {
   const organization = useOrganization();
   const location = useLocation();
-  const transaction = location.query.transaction
-    ? Array.isArray(location.query.transaction)
-      ? location.query.transaction[0]
-      : location.query.transaction
-    : undefined;
 
-  const {data: projectData} = useProjectWebVitalsQuery({transaction});
+  const {data: projectData} = useProjectRawWebVitalsQuery({});
```

Comment:
how come these don't need the transaction filter anymore?

Reply:
Thanks for the review!
`transaction` is always undefined in this scenario, so nothing should change here.
This was old branching code from when we had a transaction specific view for the `webVitalsDetailPanel` when a user clicks on a transaction/page.